src/style.css

```diff
@@ -0,0 +1,126 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 10th item. You should create separate files for each BEM-block and its elements